recipes-multimedia/alsa/imx-alsa-plugins_git.bb

```diff
@@ -20,8 +20,8 @@ inherit autotools pkgconfig use-imx-headers
 PV = "1.0.26+${SRCPV}"
 
 SRC_URI = "git://github.com/nxp-imx/imx-alsa-plugins.git;protocol=https;branch=${SRCBRANCH}"
-SRCBRANCH = "MM_04.09.02_2410_L6.6.y"
-SRCREV = "b2ba082e70333f187972ee4e85f63f9d2f608331"
+SRCBRANCH = "MM_04.09.03_2412_L6.12.y"
+SRCREV = "eaf0900662d6381bcc433165219280cca1752890"
```

Comment:
Include the list of changes between both revisions.

Reply:
Fixed